### PR TITLE
feat: SSN Demo UI fixes

### PR DESCRIPTION
**Pull request by talwinder50**
1. Logout Option
2. Tooltip
3. Dynamic DID population
4. DNS Name change from rev-agency to uscis and emp-dept to cbp
5. UI Error Handling
6. Hide email and password from UI on CBP.
Signed-off-by: talwinder.kaur <talwinder.kaur@securekey.com>
Screen Shot 2021-02-25 at 3 14 38 PM
Screen Shot 2021-02-25 at 3 15 09 PM
Screen Shot 2021-02-25 at 3 15 31 PM

### CODE REVIEW

**Review by fqutishat — APPROVED** [review]

File: cmd/acrp-rest/static/emp_dept/accountlinked.html [hunk 1]
```diff
@@ -66,6 +66,11 @@
                         <a href="/" class="text-white bold hover:bg-gray-700 hover:text-white px-3 py-6 rounded-md text-lg">Travel</a>
                     </div>
                 </div>
+                <div class="flex-1 flex items-right justify-end sm:items-stretch sm:justify-end">
```
Comment (rolsonquadras) [inline cmd/acrp-rest/static/emp_dept/accountlinked.html:69]: Don't we need to update folder names `cmd/acrp-rest/static/<mode>/accountlinked.html`  based on changes https://github.com/trustbloc/sandbox/pull/813/files#diff-7473366dd614b41c1fa9b02dd689bfa5db78bc4d5c6e8d73250294d91ce79f8aR94 ?

Comment (talwinder50) [reply cmd/acrp-rest/static/emp_dept/accountlinked.html:69]: yes i wiil do in next pr. Will be done soon before morning

Comment (rolsonquadras) [reply cmd/acrp-rest/static/emp_dept/accountlinked.html:69]: Its gonna break the master, right ?

Comment (talwinder50) [reply cmd/acrp-rest/static/emp_dept/accountlinked.html:69]: its fixed now

**Pull request merged**